libdevcore/Log.h

```diff
@@ -240,6 +229,10 @@ class LogOutputStreamBase
 		append(_t.second);
 		m_sstr << EthPurple ")" EthReset;
 	}
+	template <class T> void append(const char* _t)
```

Comment:
Not sure if we still need it in this PR, but it should be either non-template or specialization of `append` template below